src/StateSmith/Output/Sim/HtmlRenderer.cs

```diff
@@ -112,8 +131,10 @@ public static void Render(StringBuilder stringBuilder, string smName, string moc
       }
 
       .history {
-        margin-top: 30px;
-        overflow: scroll;    
+        margin-top: 30px;       
+        display: flex;
+        overflow: auto;    
+        flex-direction: column-reverse;
```

Comment:
this is such a cool trick! No js at all for perfect scroll behavior. No need for an autoscroll button. Very cool!